apps/docs/src/app/pages/table/implementation/sorting-selecting-row-actions/demos/table-1/table-one.demo.component.ts

```diff
@@ -0,0 +1,48 @@
+import { CommonModule } from '@angular/common';
+import { Component } from '@angular/core';
+import { NgxTable, NgxTableSortDirection, NgxTableSortEvent } from '@ngx/table';
+
+@Component({
+	standalone: true,
+	imports: [NgxTable, CommonModule],
+	selector: 'table-one-demo',
+	templateUrl: './table-one.demo.component.html',
+})
+export class TableOneDemoComponent {
+	currentSorting: any = undefined;
+	sortableData = [
+		{ name: 'World', firstName: 'Hello', active: true },
+		{ name: 'Tools', firstName: 'NGX', active: false },
+	];
+
+	sort(sortEvent: NgxTableSortEvent) {
```

Comment:
This is something we need to fix on project level, because it has occured a lot in PR's we've opened in the past, but try to always add a modifiers (`private`, `public`, `protected`) and return types to the properties and methods of your classes.

For example here:
```
	public currentSorting: any = undefined;
	public sortableData = [
		{ name: 'World', firstName: 'Hello', active: true },
		{ name: 'Tools', firstName: 'NGX', active: false },
	];
```
☝️ also try to provide specific types for the `currentSorting` and `sortableData` if you can.

And for the method (as an example):
```
	public sort(sortEvent: NgxTableSortEvent): void {...}
```
In this case the method doesn't return anything which is why you can type it with `: void`. Doint this implicitly, makes sure that in the future it is known that this method shouldn't return a value.

Reply:
This comment also applies to other classes. I just picked this one as an example.